src/s3fs.cpp

```diff
@@ -2808,15 +2810,15 @@ static int s3fs_statfs(const char* _path, struct statvfs* stbuf)
 #if defined(__MSYS__)
     // WinFsp resolves the free space from f_bfree * f_frsize, and the total space from f_blocks * f_frsize (in bytes).
     stbuf->f_frsize = stbuf->f_bsize;
-    stbuf->f_blocks = INT32_MAX;
-    stbuf->f_bfree = INT32_MAX;
+    stbuf->f_blocks = bucket_size;
+    stbuf->f_bfree = stbuf->f_blocks;
 #elif defined(__APPLE__)
-    stbuf->f_blocks = UINT32_MAX;
-    stbuf->f_bfree  = UINT32_MAX;
-    stbuf->f_files  = UINT32_MAX;
-    stbuf->f_ffree  = UINT32_MAX;
+    stbuf->f_blocks = bucket_size;
+    stbuf->f_bfree  = stbuf->f_blocks;
+    stbuf->f_bfree  = stbuf->f_blocks;
```

Comment:
Isn't this a mistake in `stbuf->f_files`?

Reply:
Yes, definitely. I have just pushed a new commit where f_files and f_ffree are left defined as UINT32MAX as before.